src/julia_internal.h

```diff
@@ -364,6 +365,9 @@ STATIC_INLINE jl_value_t *jl_gc_alloc_(jl_ptls_t ptls, size_t sz, void *ty)
         v = jl_gc_big_alloc(ptls, allocsz);
     }
     jl_set_typeof(v, ty);
+
+    maybe_record_alloc_to_profile(v, sz);
```

Comment:
Superflous whitespace changes

Reply:
do you think it should be nestled without any whitespace? I assume Pete was trying to keep the logging code visually separate from the code that actually does anything. Is that not idiomatic? Should we remove the extra lines?

Reply:
removed the whitespace